file/readfile.go

```diff
@@ -2,6 +2,9 @@ package file
 
 import (
 	"bufio"
+	"bytes"
+	"fmt"
+	"html/template"
```

Comment:
You probably meant `text/template`. `html/template` will perform additional escaping which I think is undesirable (will break valid YAML) here.

Please add a test ensuring that the template renderer doesn't escape yaml characters like & or <.

Reply:
Done